mobile/src/src/components/Community/Feed/EventView.jsx

```diff
@@ -67,56 +70,63 @@ const EventView = ({
             style={{ width: 0.88 * width, height: 300 }}
           />
         )}
-
         <View style={styles.buttonContainer}>
           {is_attending === true ? (
             <Button
               compact={true}
-              style={styles.buttonStyle}
+              style={styles.buttonStyleFade}
               mode="contained"
-              title="Attending"
-              color="green"
-              // todo handle press
-              // onPress={}
+              title="Unattend"
+              color="white"
+              dark={true}
+              onPress={() => {
+                attendEvent(id, !is_attending);
+              }}
             >
-              Attending
+              Unattend
             </Button>
           ) : (
             <Button
               compact={true}
               style={styles.buttonStyle}
-              mode="text"
+              mode="outlined"
               title="Attending"
-              color="green"
-              // todo handle press
-              // onPress={}
+              color="white"
+              dark={true}
+              onPress={() => {
+                attendEvent(id, !is_attending);
+              }}
             >
-              Attending
+              Attending?
             </Button>
           )}
```

Comment:
We can combine this conditional logic into 1 button that takes in props conditionally.